docs/source/getting-started/quickstart.adoc

```diff
@@ -50,17 +45,15 @@ Starting local OpenShift cluster using 'kvm' hypervisor...
 +
 [NOTE]
 ====
-- The IP is dynamically generated for each OpenShift cluster. To check
-the IP, run the `minishift ip` command.
-- By default, Minishift uses the driver most relevant to the host OS. To
-use a different driver, set the `--vm-driver` flag in `minishift start`.
-For example, to use VirtualBox instead of KVM on GNU/Linux operating
-systems, run `minishift start --vm-driver=virtualbox`. For more
-information about `minishift start` options, see the
-link:../command-ref/minishift_start{outfilesuffix}[`minishift start`] command.
+- The IP is dynamically generated for each OpenShift cluster.
+To check the IP, run the `minishift ip` command.
+- By default, Minishift uses the driver most relevant to the host OS.
+To use a different driver, set the `--vm-driver` flag in `minishift start`.
+For example, to use VirtualBox instead of KVM on GNU/Linux operating systems, run `minishift start --vm-driver=virtualbox`.
+For more information about `minishift start` options, see the xref:../command-ref/minishift_start.adoc#minishift-start[`minishift start`] command.
 ====
 
-. Use link:../command-ref/minishift_oc-env{outfilesuffix}[`minishift oc-env`] to display the command you need to type into your shell in order to add the _oc_ binary to your PATH.
+. Use xref:../command-ref/minishift_oc-env.adoc#minishift_oc-env[`minishift oc-env`] to display the command you need to type into your shell in order to add the *oc* binary to your `PATH` environment variable.
```

Comment:
I'm replying once here but it should cover all of the instances where you commented about this. In the openshift doc guidelines, the instruction is to tag "System term, GUI menu items and buttons, daemon, service, or software package" with **, while "Inline commands, operations, literal values, environment variables, parameters, settings, flags, and user input" should use ``. AFAIK a binary is a software package so that's why it gets the ** tag. When referring to env vars or commands I use ``. 

Reference: https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#quick-markup-reference